vividus-plugin-mobitru/src/main/java/org/vividus/mobitru/client/MobitruFacadeImpl.java

```diff
@@ -63,6 +63,13 @@ public MobitruFacadeImpl(MobitruClient mobitruClient)
         this.mobitruClient = mobitruClient;
     }
 
+    @Override
+    public String takeDevice(String udid) throws MobitruOperationException
```

Comment:
please merge this logic into `org.vividus.mobitru.client.MobitruFacadeImpl#takeDevice(org.openqa.selenium.remote.DesiredCapabilities)`, use `DesiredCapabilities` object to determine which API to use to take device

Reply:
done